src/annotation/Validator.java

```diff
@@ -0,0 +1,50 @@
+package annotation;
+
+import annotation.exception.ExpectedTypeException;
+import annotation.myannotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Validator {
+
+    public <T> List<InvalidData> validate(T ob) throws IllegalAccessException {
+
+        List<InvalidData> dataArrayList = new ArrayList<>();
+
+        AnnotationProcessor<T> processor;
+
+        try {
+            processor = new LengthAnnotationProcessor<>();
```

Comment:
Here you can implement the chain of responsibility design pattern,
https://www.baeldung.com/chain-of-responsibility-pattern.